Common/Net/HTTPClient.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Common/Data/Encoding/Compression.h"
 #include "Common/Net/NetBuffer.h"
 #include "Common/Log.h"
+#include "Core/HLE/sceNet.h"
```

Comment:
Why is a common (read: a helper or a utility) subsystem including a core file? This is gonna be painful to refactor later because of unexpected code dependencies.

Reply:
Yeah, please try to avoid dependencies that go in this direction.

If necessary, do move utility functions from Core to Common. If you have problems moving something, let me know and I'll figure it out.